ogr_fdw_info.c

```diff
@@ -140,11 +180,12 @@ main(int argc, char** argv)
 
 	if (err != OGRERR_NONE)
 	{
-		// printf("OGR Error: %s\n\n", CPLGetLastErrorMsg());
+		printf("OGR Error: %s\n\n", CPLGetLastErrorMsg());
+    exit(2);
 	}
 
 	OGRCleanupAll();
-	exit(0);
+	exit(1);
```

Comment:
Traditionally return code 0 means "everything OK" which is the state here, right?

Reply:
I guess that's fine. I had three states where 0 was help 1 was okay and 2 was error but no reason to not use 0 for okay. I'll fix to use 0 and 1